harmony_regridding_service/regridding_service.py

```diff
@@ -1,17 +1,4 @@
-"""Regridding service code.
-
-As of v.0.0.1 (2023-03-27) This code is written to handle only geographic
-grids and resampling.  As such there are a couple of things to pay attention
-to when we move away from this limitation.
-
-1. _is_projection_[x/y]_dim is hard coded to only care about latitudes and
-    longitudes.
-
-2. We don't worry about the coordinate metadata on the variables. Any
```

Comment:
I think is is no longer true and we need a ticket?

Reply:
that was definitely a not to myself because I didn't know what the comment actually meant and I was supposed to go find out later.